plugins/package-managers/gleam/src/main/kotlin/Gleam.kt

```diff
@@ -124,7 +124,7 @@ class Gleam internal constructor(
                     "Only the latest matching versions of direct dependencies were resolved without " +
                     "transitive dependency resolution. The results are not reproducible. " +
                     "Consider running 'gleam deps download' to generate a manifest.toml lockfile.",
-                severity = Severity.WARNING
+                severity = Severity.ERROR
```

Comment:
I guess we should then also do as usual here and omit the severity, because `ERROR` is the default?